src/lib/codemirror-editor.ts

```diff
@@ -127,6 +132,12 @@ export class CodeMirrorEditorElement extends LitElement {
       }),
       root: this.shadowRoot!,
     });
+    // EditorView writes a <style> directly into the given root on construction
+    // (unless adopted stylesheets are available in which case it uses that).
+    // But then lit renders and blows it away. So, we'll just snatch any new
+    // styles beefore this can happen, and then have lit put them back again.
+    this._capturedCodeMirrorStyles = Array.from(
+      this.shadowRoot!.querySelectorAll('style'));
```

Comment:
You don't need the `Array.from()`. querySelectorAll returns a non-live NodeList which is iterable in all the browsers we support (we don't support IE).

Reply:
Done.